hack/test-go.sh

```diff
@@ -24,6 +24,7 @@ find_test_dirs() {
         -o -wholename '*/Godeps/*' \
         -o -wholename '*/_output/*' \
         -o -wholename './.git' \
+        -o -wholename './test/extended/*' \
```

Comment:
This shouldn't to be necessary - your test cases are already guarded with build tags